Makefile

```diff
@@ -8,18 +8,15 @@ clean:
 	rm -rf build
 
 quality:
-	pep8 --config=.pep8 $(PACKAGE)
-	pylint --rcfile=pylintrc $(PACKAGE)
+	tox -e quality
 
 requirements:
 	pip install -r test_requirements.txt
 
 test:
-	nosetests --with-coverage --cover-inclusive --cover-branches \
-		--cover-html --cover-html-dir=$(COVERAGE)/html/ \
-		--cover-xml --cover-xml-file=$(COVERAGE)/coverage.xml \
-		--cover-package=$(PACKAGE) $(PACKAGE)/
+	tox
 
-validate: clean test quality
+validate:
```

Comment:
At this point, it looks like the `test` and `validate` targets are identical: they just call out to `tox`.

The `validate` target should either be updated to contain unique logic or it should probably be removed entirely.